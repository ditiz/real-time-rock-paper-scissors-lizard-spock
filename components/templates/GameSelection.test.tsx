import "@testing-library/jest-dom"
import { render, screen } from "@testing-library/react"
import GameSelectionTemplate from "./GameSelection"

it("renders GameSelectionTemplate unchanged", () => {
  const { container } = render(<GameSelectionTemplate />)
  expect(container).toMatchSnapshot()
})
