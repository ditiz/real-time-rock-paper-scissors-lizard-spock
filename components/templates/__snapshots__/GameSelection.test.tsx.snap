// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders GameSelectionTemplate unchanged 1`] = `
<div>
  <nav
    class="navbar"
  >
    <div
      class="navbar__logo"
    >
      <img
        alt="logo"
        data-nimg="1"
        decoding="async"
        height="25"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=64&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=64&q=75 2x"
        style="color: transparent;"
        width="25"
      />
    </div>
    <div
      class="navbar__links"
    >
      <a
        href="/"
      >
        Home
      </a>
      <a
        href="/Rules"
      >
        Rules
      </a>
      <a
        href="/Classic"
      >
        Classic
      </a>
      <a
        href="/Advanced"
      >
        Advanced
      </a>
    </div>
  </nav>
  <h1>
    Game selection
  </h1>
  <section
    class="game-selection"
  >
    <article
      class="card"
    >
      <a
        href="/Classic"
      >
        <div>
          <h2>
            Game Classic
          </h2>
          <p>
            Classic game of rock, paper, scissors
          </p>
        </div>
      </a>
    </article>
    <article
      class="card"
    >
      <a
        href="/Advanced"
      >
        <div>
          <h2>
            Game Advanced
          </h2>
          <p>
            Game of rock, paper, scissors, lizard, spock
          </p>
        </div>
      </a>
    </article>
  </section>
</div>
`;
